main/dzVents.cpp

```diff
@@ -93,8 +93,8 @@ void CdzVents::ProcessSecurity(lua_State *lua_state, const std::vector<CEventSys
 void CdzVents::ProcessHttpResponse(lua_State *lua_state, const std::vector<CEventSystem::_tEventQueue> &items)
 {
 	int index = 1;
-	int statusCode;
-
+	std::string status = "" ;
```

Comment:
I would split the string into a status_code (integer) and a status_text (string) below